mxd/README.md

```diff
@@ -699,6 +699,279 @@ and the proxied URL will be `https://jsonplaceholder.typicode.com/todos/1`.
 
 ## 7. Add new participant Trudy
 
+In our current setup, there are two connectors named alice and bob. Now we will be adding a new connector named trudy.  
+Following are the steps needed to accomplish this.
+
+### 7.1 Create Keycloak Client
+Currently, Alice and Bob each have a keycloak client name `alice-private-client` and `bob-private-client`.  
+We need to create a client for Trudy as well.  
+A client named `trudy-private-client` has already been created.
```

Comment:
This does not make sense. Either we need to create a client, or it already has been created.